.travis.yml

```diff
@@ -14,6 +14,8 @@ before_install:
 
 install:
   - yarn
+  - export DISPLAY=':99.0'
+  - Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
```

Comment:
those configuration are already defined (see services->xvfb) under linux tests.
i don't think we need this commit